website/addons/s3/requirements.txt

```diff
@@ -1 +1 @@
-boto==2.31.1
+boto==2.38.0
```

Comment:
Not sure what happened to my previous comment, but did we come to a resolution on whether we can make this upgrade or not? @HarryRybacki @icereval